src/controllers/createGameController.ts

```diff
@@ -0,0 +1,14 @@
+import { CreateGameInteractor } from "../Interactors/createGameInteractor";
+
+export class CreateGameController {
+    private readonly useCase : CreateGameInteractor
+
+    constructor(useCase : CreateGameInteractor) {
+        this.useCase = useCase;
+    }
+
+    execute() {
+        return this.useCase.execute();
+        //var chosenWord = this.useCase.execute();
```

Comment:
Nepalik žodinių komentarų(code must be self explanatory) ir užkomentuoto kodo(jei buvo parašytas ir commited bet jo nebereikia, tai prisireikus gali rasti per git history, o jei parašei ir užkomentavai be commit, tai greičiausiai kad jo niekada ir nereikės)